lib/ansible/modules/cloud/amazon/ec2_vpc_igw.py

```diff
@@ -7,37 +7,48 @@
 
 
 ANSIBLE_METADATA = {'metadata_version': '1.1',
-                    'status': ['stableinterface'],
-                    'supported_by': 'certified'}
+                    'status': ['preview'],
```

Comment:
We can't change these

Reply:
Reverted the change in latest commit. Please review and let me know if you have any suggestions.